src/components/Dashboard/TodoList/PaymentPendingOnlineBanking.vue

```diff
@@ -22,7 +22,7 @@
 import { Component, Vue, Prop } from 'vue-property-decorator'
 
 @Component({})
-export default class OnlineBankingPaymentPending extends Vue {
```

Comment:
What can I say? I like consistency...

![image](https://user-images.githubusercontent.com/10002889/104348067-26ffa000-54b6-11eb-8d79-28862647cbec.png)

Reply:
i like it!